classes/class-p4-master-site.php

```diff
@@ -783,4 +784,24 @@ public function gdpr_cc_comment_form_add_class( $submit_field, $args ) {
 
 		return $submit_field;
 	}
+
+	/**
+	 * Filter function for embed_oembed_html.
+	 * Transform youtube embeds to youtube-nocookie.
+	 *
+	 * @see https://developer.wordpress.org/reference/hooks/embed_oembed_html/
+	 *
+	 * @param $cache mixed  The cached HTML result, stored in post meta.
+	 * @param $url   string The attempted embed URL.
```

Comment:
I think these should be reversed, like `@param string $url` and phpcs is complaining.